Add basic tests

wesleybl opened the pull request:
Also separate profile uninstall from Plone 5 and Plone 4. Plone 4 has no resources to remove.

mauritsvanrees reviewed (approved):
Seems about right from a quick glance.
`uninstall-base` should be added to the hidden profiles.
I wonder if it may be less confusing though to make a Plone 5-only branch.

To be honest, I don't really care about this package, so am not that interested in reviewing.

I will happily make a release though, or grant you PyPI rights.

wesleybl commented:
@mauritsvanrees I added `collective.js.jqueryui:uninstall` and `collective.js.jqueryui:uninstall-base` to the hidden profiles and added a test to check this.

Thanks for the review.
